extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisReceiver.scala

```diff
@@ -82,15 +82,19 @@ private[kinesis] class KinesisReceiver(
   var workerId: String = null
 
   /*
-   * This impl uses the DefaultAWSCredentialsProviderChain and searches for credentials 
-   *   in the following order of precedence:
+   * This impl uses the DefaultAWSCredentialsProviderChain unless it's provided by constructor
+   *  and searches for credentials in the following order of precedence:
```

Comment:
might be good to reword this to be more explicit about the 2 different scenarios:
1) Some[AWSCredentials] are provided in which case we use those
2) None are provided and therefore DefaultAWSCredentialsProviderChain will be constructed which searches for credentials in the following order...